chrome/content/zotero/xpcom/collectionTreeView.js

```diff
@@ -1850,8 +1927,25 @@ Zotero.CollectionTreeView.prototype.drop = Zotero.Promise.coroutine(function* (r
 			return;
 		}
 		
+		var items = yield Zotero.Items.getAsync(ids);
+		if (!items) {
```

Comment:
`items` is only possibly false if an integer is passed to `getAsync()`. If an array is passed, it would be an empty array.